debian/changelog

```diff
@@ -0,0 +1,23 @@
+gst-plugin-pylon (0.3.0-2) unstable; urgency=medium
```

Comment:
Please use the changelog in the root as a reference.

Reply:
I'm sorry, I'm not certain I understand how to make this change.

Reply:
See the [ChangeLog](https://gitlab.com/RidgeRun/customers/basler/gst-plugin-pylon/-/blob/main/CHANGELOG.md) we have in the repository already. That is the information that should be in this ChangeLog as well. It needs to be formatted to meet Debian format requirements though.

Reply:
The changelog will not list the commits, but high level changes. For example, instead of: 
```
   * Removed unused auto generated Debian packaging files and comments
   * Updated the Debian packaging contact information and license to match those from Basler
   * Changed the Debian control file 'Section' value to 'libs'
```
we will have your Debian changes in the next release as:
```
   * Added support for building the project as a Debian package.
```

Having said that, the changelog as per now should look like:
```
gst-plugin-pylon (0.3.0-0) unstable; urgency=medium

   * Added - Support to access stream grabber parameters via stream:: prefix
   * Added - Bayer 8bit video data support
   * Added - Add property to configure camera from a pylon feature stream file ( pfs-location=<filename> )
   * Added - Support to build gst-plugin-pylon as monorepo subproject to test against upcoming changes in gstreamer monorepo
   * Added - Add pygobject example to access camera and stream grabber features in the folder examples
   * Changed - Remove fixed grab timeout of 5s. pylonsrc will now wait forever. If required timeout/underrun detection can be handled in the gstreamer pipeline. ( e.g. via queue underrun )
   * Changed - Remove limititation to only map features that are streamable Effect is that now all accessible features of a camera are mapped to gstreamer properties
   * Changed - Output more information in case of missing selection of a camera. Now: serial number, model name and user defined name.
   * Changed - Run all tests in github actions
   * Fixed - Allow loading of default user set on some early Basler USB3Vision cameras

  -- Thies Möller thies.moeller@baslerweb.com  Tue, 13 Sep 2022 00:00:00 -0700

 gst-plugin-pylon (0.2.0-0) unstable; urgency=medium

   * Added - Initial public release

  -- Thies Möller thies.moeller@baslerweb.com  Tue, 09 Aug 2022 00:00:00 -0700
```

Your Debian changes will be included in the next release 0.4.0-0 release, hopefully